tests/test_validators_cerberus.py

```diff
@@ -0,0 +1,299 @@
+from __future__ import absolute_import
+import pytest
+
+from spidermon.contrib.validation import CerberusValidator
+from spidermon.contrib.validation import messages
+
+# Format for parameters
+# 1. schema
+# 2. data
+# 3. valid
+# 4. expected_errors
+# 5. Id of test
+
+# Format of test
```

Comment:
Here you are just copying the implementation code. Remove this comment. The parameters and the code is self-explanatory. No need for this kind of comment here.